css/style.css

```diff
@@ -0,0 +1,64 @@
+
+html {
+    font-size: 10px;
+}
+
+body
+{
+    background-color: #dcdcdc;
+    text-align: center;
+}
+
+h1
+{
+    font-size: 5rem;
+    margin-bottom: 0;
+}
+
+
+#img-div
+{
+    background: white;
+}
+
+#image
+{
+    max-width: 100%;
+    display: block;
+    margin: auto;
+    margin: 0 auto;
+}
+
+a {
+    color: #477ca7;
+}
+a:visited {
+    color: #74638f;
+}
+
+ul {
+    max-width: 550px;
+    margin: 0 auto 50px auto;
+    text-align: left;
+    line-height: 1.6;
+}
+
+
+ul {
+    max-width: 550px;
+    margin: 0 auto 50px auto;
+    text-align: left;
+    line-height: 1.6;
+}
+
+li {
+    margin: 20px 0;
+}
+
+
```

Comment:
Nice job finding this duplicated code in the project! I have a suggestion to make your comment even better:
- Could you please provide some context or explain **why** duplicating code is a bad idea? This would help the student better understand your perspective and evaluate the suggestion. 💪

Reply:
Thanks i will do that